### DISCUSSION TITLE
Remove loot card 1418 from the lootdeck

### DISCUSSION THREAD

**Discussion by kimmathiassen** [Lurkars/gloomhavensecretariat#449] [Q&A] [answered]
I am having a hard time figureing out how to remove loot card 1418 from the loot deck. I my game I read section 133.4 where i was instructed to remove the lootcard, but I dont know how to do that in this app. 
Is there a way to remove the lootcard?

The app has no way of knowing which section i selected from the lootcard, if there is no implementation to support the removal of this card then I suggest that it is added to section 80.4, which happens one week after the lootcard was removed. 

Cheers

**Comment by Lurkars** [comment, marked as answer]
Yes, marking the sections and therefore automatically remove card was added just a few releases ago.

Anyway, you can remove the card manually by editing the loot deck:

- open the loot deck menu (clicking on the settings icon next to it or on a drawn card)
- toggle edit mode
- click on *Show Loot Table*
- scroll down and remove special card

**Reply by kimmathiassen** [reply]
I think I figured it out. 
I was a bit surprised that I had to be in a scenario before I could edit the lootdecks global state, to remove the special loot cards.
Thanks for the help, cheers.

**Reply by Lurkars** [reply]
Ah good point, outside scenario only enhancements are editable right. I will add toggle for the special loot cards there too...
